1.train_models/log_reg_plates_3_3p_5_cp_norm_data/nbconverted/plate_3_3p_5_train_log_reg_scv.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Random Search with logistic regression (Genotype Classification)
+# We perform a random search using logistic regression to improve the classification performance on plates 3, 3 prime, and 5.
+
+# In[1]:
+
+
+import pathlib
+import random
+import sys
+import warnings
+from collections import defaultdict
+
+import pandas as pd
+from joblib import dump
+from sklearn.exceptions import ConvergenceWarning
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import StratifiedKFold
+from sklearn.preprocessing import LabelEncoder
+from sklearn.utils import parallel_backend
+
+# ## Find the root of the git repo on the host system
+
+# In[2]:
+
+
+# Get the current working directory
+cwd = pathlib.Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
+
+# Check if a Git root directory was found
+if root_dir is None:
+    raise FileNotFoundError("No Git root directory found.")
```

Comment:
I've thought about this, but then I would have to specify a relative path to this new module, in which case, It would be easier to just to specify the relative path to the root git directory and avoid this using this code.